chart/values.yaml

```diff
@@ -306,6 +306,8 @@ ingress:
   tls: false
 
 ingress_class: nginx
+# Optional: Uncomment to use your own cluster-issuer instead of default ACME https validation
```

Comment:
Could you move cluster-issuer property to be under the `ingress` map above to make it more clearer it only applies if ingress is used? And perhaps name it `custom_cluster_issuer` to be even more clear its for custom use only.

Reply:
do you mean to become child of ingress map and thus ".Values.ingress.custom_cluster_issuer" or just move the two lines up to 307?

Reply:
Yes, part of ingress map and `.Values.ingress.custom_cluster_issuer`. Thanks!